Source/DafnyDriver/DafnyNewCli.cs

```diff
@@ -218,7 +222,7 @@ public static class DafnyNewCli {
     }
     var projectFile = await DafnyProject.Open(OnDiskFileSystem.Instance, dafnyOptions, file);
 
-    projectFile.Validate(dafnyOptions.OutputWriter, AllOptions);
+    projectFile.Validate(dafnyOptions.OutputWriter, AllOptions.Concat(DafnyProject.targetOptions));
```

Comment:
Also part of the extra feature of target-language-specific project options, so I suggest leaving for a later PR.

Reply:
Answered above, removed all project files related stuff. Actually moved them here: https://github.com/ShubhamChaturvedi7/dafny/tree/GoLibrayModuleSupport_WithDafnyProject